src/image-default.yaml

```diff
@@ -22,3 +22,4 @@ squashfs-compression: zstd
 # Defaults for VMware OVA, matching historical behavior
 vmware-hw-version: 13
 vmware-os-type: rhel7_64Guest
+vmware-secure-boot: "true"
```

Comment:
Er, ideally we'd make this an actual boolean, not a string of one.

Reply:
@mike-nguyen

Reply:
The boolean gets converted to `True` when it makes it into the `coreos.ovf` file which  wasn't consistent with the rest of the file. If there's no problem with that I will change it to an actual boolean.  I didn't have access to vmware to test.

```
$ cat coreos.ovf | grep -i -e false -e true
      <Item ovf:required="false">
        <rasd:AutomaticAllocation>false</rasd:AutomaticAllocation>
        <vmw:Config ovf:required="false" vmw:key="allowUnrestrictedCommunication" vmw:value="false"/>
        <vmw:Config ovf:required="false" vmw:key="backing.writeThrough" vmw:value="false"/>
        <rasd:AutomaticAllocation>true</rasd:AutomaticAllocation>
        <vmw:Config ovf:required="false" vmw:key="connectable.allowGuestControl" vmw:value="true"/>
        <vmw:Config ovf:required="false" vmw:key="wakeOnLanEnabled" vmw:value="false"/>
      <vmw:Config ovf:required="false" vmw:key="bootOptions.efiSecureBootEnabled" vmw:value="true"/>
      <vmw:Config ovf:required="false" vmw:key="firmware" vmw:value="efi"/>
      <Property ovf:userConfigurable="true" ovf:type="string"
      <Property ovf:userConfigurable="true" ovf:type="string"
```

Reply:
Thanks for the feedback @bgilbert!  Thinking about your comment more now, the way I did it makes it more confusing for people who are setting values in `image*.yaml`.  I opened these PRs to fix it:

https://github.com/coreos/coreos-assembler/pull/2980
https://github.com/openshift/os/pull/892